assets/eip-5008/package.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "EIP-5006",
```

```suggestion
    "name": "EIP-5006",
    "dependencies": {
        "@openzeppelin/contracts": "^4.7.3"
    },
```
Recommended: Add OZ to package.json